### PR TITLE
[luci] Add test for quantization with layer info

### PR DESCRIPTION
**Pull request by jinevening**
This adds test for quantization with layer info.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/pull/8558#issuecomment-1057792092

### CODE REVIEW

File: compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp [hunk 1]
```diff
@@ -244,6 +244,7 @@ struct InsertQuantizeOp final : public luci::CircleNodeMutableVisitor<void>
   INSERT_QUANTIZE_TO_BINARY_OP(luci::CircleMinimum, x, y)
   INSERT_QUANTIZE_TO_BINARY_OP(luci::CircleMul, x, y)
   INSERT_QUANTIZE_TO_BINARY_OP(luci::CircleNotEqual, x, y)
+  INSERT_QUANTIZE_TO_BINARY_OP(luci::CircleOneHot, on_value, off_value)
```
Comment (jinevening) [inline compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:247]: This code is required to add test for OneHot op.

**Comment by seanshpark** [comment]
There's a `Type` type that uses general word `Type` which is confusing.
I assume this is data type and also I may have agreed as this has landed.
But I highly recommend to change this not `Type` but something else.

**Comment by jinevening** [comment]
`Type` is alias of `loco::DataType` in this file. Renaming will be done in the next PR.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**